mavproxy/source/docs/modules/systemtime.rst

```diff
@@ -23,3 +23,4 @@ interval_timesync    How often to send a timesync message (seconds)   10
 interval             How often to send the system time (seconds)      10
 ==================   ===============================================  ===============================
 
+Note: you will need to set the `BRD_RTC_TYPES <https://ardupilot.org/copter/docs/parameters.html#brd-rtc-types-allowed-sources-of-rtc-time>`__ = 2 on the autopilot to enable updates from ``SYSTEM_TIME`` messages.

```

Comment:
Hmmm.  This is a bitmask, so setting it to 2 - while it will work - isn't necessarily the value the user wants.


```suggestion
Note: you will need to include the value "2" (MAVLINK_SYSTEM_TIME) in the `BRD_RTC_TYPES <https://ardupilot.org/copter/docs/parameters.html#brd-rtc-types-allowed-sources-of-rtc-time>`__ bitmask on the autopilot to enable updates from ``SYSTEM_TIME`` messages.

```

Is that still clear enough?

Reply:
Hi Peter, yes I think that's still clear enough.